asyncstatsd/metrics/basic.py

```diff
@@ -0,0 +1,61 @@
+from dataclasses import dataclass, field
+from datetime import timedelta
+
+
+@dataclass
+class StatsdMetric:
+    name: str
+    value: int | float | timedelta
+    unit: str
+    rate: float = field(default=1.0)
+
+    def get_value(self) -> str:
```

Comment:
Minor, but I find it unintuitive that `value` and `get_value()` have different types (numeric vs string). Would it make sense to reflect the string nature of the return value in the method name? e.g. `get_value_str()`.

Reply:
noted

Reply:
fixed